app/retail/templates/gas_faq.html

```diff
@@ -12,8 +12,6 @@
 
 
 <h1>{% trans "FAQ" %}</h1>
-<a href="{% url 'gas_faucet_list' %}" target="_blank" rel="noopener noreferrer">{% trans "I need gas money, where can I get it?" %}</a>
```

Comment:
do we need to take this out? it's a valid faq... maybe point to https://faucet.metamask.io/